packages/core/src/external/carequality/dq/process-inbound-dq.ts

```diff
@@ -64,6 +58,26 @@ export async function processInboundDocumentQuery(
   }
 }
 
+async function getDocumentContents(
+  payload: InboundDocumentQueryReq,
+  apiUrl: string,
+  cxId: string,
+  patientId: string
+): Promise<string[]> {
+  let documentContents = await findDocumentReferences(payload);
+  if (!documentContents) {
+    const endpointUrl = `${apiUrl}/internal/docs/ccd`;
+    await createAndUploadCcdAndMetadata(cxId, patientId, endpointUrl);
+    documentContents = await findDocumentReferences(payload);
+  }
+  if (!documentContents) {
+    const msg = `Error getting document contents`;
+    capture.error(msg, { extra: { cxId, patientId } });
+    throw new XDSRegistryError("Internal Server Error");
+  }
```

Comment:
Don't we just want to return "no docs found" in this case? Could alert us, for sure but it shouldn't fail, right?